content/control-api/curl-examples.textile

```diff
@@ -0,0 +1,205 @@
+---
+title: Examples
+meta_description: "The Control API is a REST API that enables you to manage your Ably account programmatically."
+meta_keywords: "Ably, Control API, account management, control, REST API, OpenAPI Specification, OAS3."
+section: control-api
+index: 20
+jump_to:
+  Help with:
+    - Examples - Apps#examples-apps
+    - Examples - Queues#examples-queues
+    - See also#see-also
+---
+
+This topic includes some examples of using the Control API with "Curl":https://curl.se/.
+
+In the code examples, you will need to set the following variables by any convenient method (such as setting the variables in a script, or copying and pasting suitable values directly into the code):
+
+|_. Variable |_. Description |
+| ACCOUNT_ID | Your Ably Account ID (see "here":/control-api#account-id) |
+| ACCESS_TOKEN | Your Ably access token for the Control API (see "here":/control-api#authentication) |
+| APP_ID | The application ID for the application you want to modify or retrieve information for (see "here":/control-api#app-id) |
+
+h2(#examples-apps). Applications
+
+You can use the Control API to carry out tasks such as listing and creating applications. Operations available include:
+
+* Get a list of apps
+* Create an app
+* Update an app
+* Delete an app
+* Update an app's APNS info 
```

Comment:
I think it would be good to link [here](https://developer.apple.com/library/archive/documentation/NetworkingInternet/Conceptual/RemoteNotificationsPG/APNSOverview.html), or somewhere else that explains what APNS is (I had to Google it 😊 )